precheck.bash

```diff
@@ -16,7 +16,7 @@ TEST_HYDRO=false
 TEST_HYDRO_CLI=false
 TEST_ALL=false
 
-while (( $# )) do
+while (( $# )); do
```

Comment:
bug before or after??

Reply:
Bug before, seems certain bash accepted the previous syntax but the fix is more correct